packages/hyperion-autologging/src/ALIReactFlowlet.ts

```diff
@@ -41,12 +41,12 @@ export function init(options: InitOptions) {
     IReactModule.useLayoutEffect,
     IReactModule.useReducer
   ].forEach(fi => {
-    fi.onArgsMapperAdd(args => {
+    fi.onBeforeCallArgsMapperAdd(args => {
       args[0] = flowletManager.wrap(args[0], fi.name);
       return args;
     });
   });
-  IReactModule.useState.onValueMapperAdd(value => {
+  IReactModule.useState.onAfterReturnValueMapperAdd(value => {
```

Comment:
A few thoughts on the naming pattern:
- Suggest changing the order from `...Add` to `add...`, to follow similar patterns as `addEventListener`
- Suggest using the same `...BeforeX...`/`...AfterX...` phrase 

So maybe renaming to something like `addArgsMapperBeforeCall` and `addValueMapperAfterCall`?